app/public/templates/scene.html

```diff
@@ -93,10 +101,29 @@
           {% end %}
           fov="80"
           {% if scene_dict["camera_properties"]["look_controls"] %}
-            look-controls
+            {% if cursor_mode == 2 %}
+              {% if cur_scene_idx == 0 %}
+                look-controls="enabled: false; pointerLockEnabled: true;"
+              {% else %}
+                look-controls="enabled: true; pointerLockEnabled: true;"
+              {% end %}
+            {% else %}
+              look-controls
+            {% end %}
           {% end %}
           position="0 {{ scene_dict['camera_properties']['position'][1] }} 0"
         >
+        {% if not cursor_mode == 1 %}
```

Comment:
I assume this can be simplified to cursor_mode == 2 or cursor_mode == 3 , since we only support 3 cursor_modes in total?